3-Authorization-II/2-call-api-b2c/API/authConfig.js

```diff
@@ -0,0 +1,33 @@
+const passportConfig = {
+    credentials: {
+        tenantName: 'fabrikamb2c.onmicrosoft.com',
+        clientID: 'e29ac359-6a90-4f9e-b31c-8f64e1ac20cb',
+    },
+    policies: {
+        policyName: 'B2C_1_susi',
```

Comment:
```suggestion
        policyName: 'B2C_1_susi',
```

Need to highlight this in README that they should replace this with their own user-flow